### DISCUSSION TITLE
Security talent "Extra Powder" does not effect depth charges

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#14440] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If a character has maxed out weapons skill and the talent Extra Powder you would expect any explosives they craft to be masterwork. Instead all forms of depth charge (nuclear and decoy included) only craft at excellent quality. 

Depth charges are described as "barrels of explosives" and have both explosion radius and explosive damage, so this really seems like depth charges should be considered a form of explosive.

### Reproduction steps

start a singleplayer campaign
obtain a maxed out security officer with Extra Powder
make a depth charge of any kind in a fabricator
see that it is only excellent quality

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
No error messages.
```

**Comment by Regalis11** [comment]
Thank you for the report! Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/51d41ebeddc6d729e8bf0ac191bed287515c89ea (commit in our private repo), fix coming up in the next update.
